packages/core/src/tools/linking.helper.ts

```diff
@@ -114,3 +114,18 @@ export const createAndroidNavigaitonURLParams = (
     {param: 'mode', value: transportType},
   ];
 };
+
+export const createGoogleMapsDirectionsURL = (
+  pois: Array<{address: string; latitude?: number; longitude?: number}>,
```

Comment:
I think address is not required as user can choose between address and coordinates